src/storage.js

```diff
@@ -23,4 +26,41 @@ module.exports = {
   getFeeds: () => redis.smembers(FEEDS),
 
   getFeed: feedID => redis.hgetall(feedID),
+
+  //  addPost
+  addPost: async post => {
+    const postId = await redis.incr(POST_ID);
+    await redis
+      .multi()
+      .hmset(
+        postId,
+        'guid',
+        post.guid,
+        'author',
+        post.author,
+        'title',
+        post.title,
+        'link',
+        post.link,
+        'content',
+        post.content,
+        'text',
+        post.text,
+        'updated',
+        post.updated,
+        'published',
+        post.published,
+        'url',
+        post.url,
+        'site',
+        post.site
+      )
+      .zadd(POSTS, Date.parse(post.published), postId)
+      .exec();
+    return postId;
```

Comment:
I don't think we need to return anything, since we'll already know the key for this post, namely, the `post.guid`